### ISSUE TITLE
Feature Request:  Customizable/UserFriendly weather patterns.

### ISSUE THREAD

**Issue by tankfest** [project-topaz/topaz#837]
**_I have:_**

- [x] searched existing issues (http://project-topaz.com/issues/) to see if the issue has already been opened
- [x] checked the commit log to see if the issue has been resolved since my server was last updated

**_Additional Information_:** 

Kreidos's overhaul for zone_weather provided fantastic optimization for the server, but it eliminated ability to customize weather for each zone.  Before the optimizations, customizing weather was a bit of a chore, but still possible.  Can we provide the best of both worlds somehow?

**Comment by kaincenteno** [comment]
Maybe a weather generator script would help with that?

**Comment by tankfest** [comment]
Agreed with that, possibly able to package it with topaz or added as a resource?

**Comment by Kreidos** [comment]
The previous weather table meant editing 2160 SQL rows if you wanted to change 1 zone's weather. I'd argue it was no more user-friendly before, ha ha.

But the new format is well documented and easy to work with through scripting / python. A weather tool was something I hoped might eventually be made.
